docker-compose.yml

```diff
@@ -52,3 +55,38 @@ services:
       POSTGRES_USER: algorand
       POSTGRES_PASSWORD: algorand
       POSTGRES_DB: indexer_db
+
+  algosearch:
+    container_name: "algorand-sandbox-algosearch"
+    build:
+      context: .
+      dockerfile: ./images/search/Dockerfile
+      args:
+        URL: "${ALGOSEARCH_URL}"
+        BRANCH: "${ALGOSEARCH_BRANCH}"
+        SHA: "${ALGOSEARCH_SHA}"
+    ports:
+      - 8000:8000
+    environment:
+      DISABLED: "${INDEXER_DISABLED}"
```

Comment:
can you please use a different flag for AlgoSearch?

Reply:
Done.